src/components/MarkdownViewer.tsx

```
@@ -110,167 +110,212 @@ const MarkdownViewer: React.FC<MarkdownViewerProps> = React.memo(({ pdfUrl }) =>
+  }
+
+  const handleApplyAllSuggestions = async () => {
+    if (applyAllCountRemaining <= 0) {
+      failToast("Kesempatan untuk apply all saran sudah habis!")
+      return
+    }
+    setApplyAllCountRemaining(0)
+    setIsApplying(true)
+    let successCount = 0
+    let failCount = 0
+    let failedTitles: string[] = []
```

ESLint

[error] 'failedTitles' is never reassigned. Use 'const' instead.

(prefer-const)